embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/comms/delivery/FailedApiCallsPerEndpoint.kt

```diff
@@ -0,0 +1,80 @@
+package io.embrace.android.embracesdk.comms.delivery
+
+import io.embrace.android.embracesdk.comms.api.EmbraceApiService.Companion.Endpoint
+import java.util.concurrent.ConcurrentHashMap
+
+/**
+ * A map containing a list of failed API calls per endpoint.
+ */
+internal class FailedApiCallsPerEndpoint {
```

Comment:
I will do the rename in another PR so I don't complicate the review of this PR and also because we need to modify other components to be consistent on the naming we give to these Api Calls.